src/output.rs

```diff
@@ -158,8 +158,9 @@ impl Scale {
     }
 }
 
+/// Inner output data
 #[derive(Debug)]
-pub(crate) struct Inner {
+pub struct Inner {
```

Comment:
Feel free to disagree, but I'm not sure if exposing internal types is a good idea, and will look quite weird in the docs, but I understand that it removed one level of nested structs, so I suppose I'm fine with it (not really, but I'm trying to not be too biased :smile: ).

Maybe we could at least pick a less internally sounding type name, not sure what that would be tho, `OutputDataInner`?

Reply:
Yep, OutputData struct initially was only meant to be UserData type wrapper so your change to move it to wayland module if for sure the right decision.